enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabaseLinksToggleDescription.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
```

Comment:
Not sure why it said this is a totally new file. You can see the old file here for comparison.
https://github.com/metabase/metabase/blob/master/enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabaseLinksToggleWidget/MetabaseLinksToggleDescription.tsx